README.md

```diff
@@ -5,6 +5,28 @@
 
 ## Build Instructions
 
+### Condensed Steps
+
+Please refer to the sections below for more detailed information.
+
+1. Download and install Xcode.
+1. `git clone git@github.com:wordpress-mobile/WordPress-iOS.git` in the folder of your preference.
+1. `cd WordPress-iOS` to enter the working directory.
+1. `rake dependencies` this may take some time to complete.
+1. `rake xcode` to launch the workspace.
+1. Compile and run the app on a device or an emulator.
+
+In order to login to WordPress.com using the app:
+
+1. Create a WordPress.com account at https://wordpress.com/start/user (if you don't already have one).
+1. Create an application at https://developer.wordpress.com/apps/ 
```

Comment:
Nitpick: This is the only step that does not end with a period. 🙂

Reply:
Hah. That's because I have the habit of leaving punctuation off of sentences that end with a URL… I will fix.